docs/delayed_embed.md

```diff
@@ -81,9 +90,10 @@ DECLARE
   api_key text;
 BEGIN
   api_key := config->>'api_key';
-  FOR r IN select id, contents from document_embedding where embedding IS NULL LOOP
+  FOR r IN select id, contents from document_embedding where embedding IS NULL FOR UPDATE SKIP LOCKED LOOP
```

Comment:
I think you'll want to use `LIMIT` and lock a batch of N at a time. Loop until there are none left to do. This limits the number of rows that get locked at the same time.

Reply:
Should we limit 1 by 1?